main_csv.py

```diff
@@ -0,0 +1,58 @@
+import re
+import csv
+
+
+def get_data():
+
+    os_prod_list = []
+    os_name_list = []
+    os_code_list = []
+    os_type_list = []
+    main_data = []
+
+    for i in range(1, 4):
+        file_obj = open(f'info_{i}.txt')
+        data = file_obj.read()
+
+        # Получаем список изготовителей системы
+        os_prod_reg = re.compile(r'Изготовитель системы:\s*\S*')
+        os_prod_list.append(os_prod_reg.findall(data)[0].split()[2])
+
+        # Название ОС
+        os_name_reg = re.compile(r'Windows\s\S*')
+        os_name_list.append(os_name_reg.findall(data)[0])
+
+        # Код продукта
+        os_code_reg = re.compile(r'Код продукта:\s*\S*')
+        os_code_list.append(os_code_reg.findall(data)[0].split()[2])
+
+        # Тип системы
+        os_type_reg = re.compile(r'Тип системы:\s*\S*')
+        os_type_list.append(os_type_reg.findall(data)[0].split()[2])
+

```

Comment:
Ideally, there should be support for missing values (like Product code) and, optionally, English language.
![image](https://user-images.githubusercontent.com/2947151/87873553-38666480-c9cb-11ea-84f1-49db7bad2cfa.png)
[info_4.txt](https://github.com/artpetrovskii/Client-server-applications-in-python/files/4943609/info_4.txt)